server/graphql/v2/input/NewPaymentMethodDataInput.js

```diff
@@ -0,0 +1,14 @@
+import { GraphQLInputObjectType, GraphQLInt, GraphQLNonNull, GraphQLString } from 'graphql';
+
+export const NewPaymentMethodDataInput = new GraphQLInputObjectType({
```

Comment:
According to the naming rules. https://github.com/opencollective/opencollective/issues/2935 It should be: `PaymentMethodCreateInput`